code_to_gast/helpers/py_helpers.py

```diff
@@ -54,6 +54,19 @@ def array_to_gast(node):
     gast["elts"] = list_elem
     return gast
 
+"""
+takes a dictionary converts it for the generic AST 
+"""
+def dictionary_to_gast(node):
+    gast = {"type": "dict", "elts": []}
```

Comment:
Are we already using elts? If we are we can keep it for consistency otherwise I'd prefer `elts` --> `elements` or `items`

Reply:
+1

Reply:
We have had them as elts for lists and dictionaries but we can change if you think that makes sense

Reply:
Just updated this on the design doc and in the code base.